CMake/Packages.cmake

```diff
@@ -39,7 +39,7 @@ elseif(LINUX)
     set(PACKAGE_DEPENDENCIES
       "glibc >= 2.12"
       "openssl >= 1.0"
-      "device-mapper >= 1.02.90"
+      "device-mapper >= 7:1.02.90"
```

Comment:
This will apply to all CentOS/Oracle/RHEL/Amazon-based versions. We'll want to remove device-mapper from the enterprise linux list and apply this "epoch-included" version string based on the epoch: 6 or 7 right?